numpy/lib/npyio.py

```diff
@@ -599,13 +611,41 @@ def savez_compressed(file, *args, **kwds):
     """
     _savez(file, args, kwds, True)
 
+@contextlib.contextmanager
```

Comment:
Right, sorry, I don't know why I'm having so much trouble putting my comments on the right line.